app/assets/javascripts/discourse/app/widgets/do-not-disturb.js

```diff
@@ -0,0 +1,46 @@
+import I18n from "I18n";
+import { createWidget } from "discourse/widgets/widget";
+import { h } from "virtual-dom";
+import { iconNode } from "discourse-common/lib/icon-library";
+import { publishDoNotDisturbOffFor } from "discourse/lib/do-not-disturb";
+import showModal from "discourse/lib/show-modal";
+
+export default createWidget("do-not-disturb", {
+  tagName: "div.btn.do-not-disturb-btn",
+
+  html() {
+    if (this.currentUser.do_not_disturb_until) {
```

Comment:
I might be mistaken but it seems this widget is only attached if `do_not_disturb_until` is true so this if statement would always be true?

Reply:
Nah, this is always rendered on the notifications pane.